src/app/layout/qgslayoutappmenuprovider.cpp

```diff
@@ -38,7 +38,7 @@ QMenu *QgsLayoutAppMenuProvider::createContextMenu( QWidget *parent, QgsLayout *
   //undo/redo
   menu->addAction( layout->undoStack()->stack()->createUndoAction( menu ) );
   menu->addAction( layout->undoStack()->stack()->createRedoAction( menu ) );
-  menu->addSeparator();
+  menu->addSeparator()->setText( QLatin1String( "UndoRedoSeparator" ) );
```

Comment:
I think this should be:

```suggestion
  menu->addSeparator()->setObjectName( QLatin1String( "UndoRedoSeparator" ) );
```